sys-boot/udk/files/udk-workspace.template

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+show_usage() {
+	if [ ! -z "${1}" ]; then
+		echo "${1}" >&2
+		echo >&2
+	fi
+	echo "Usage:"
+	echo ". ${BASH_SOURCE[0]} [folder]"
+	echo "where folder is the workspace folder - current directory is used if unspecified"
+}
+
+create_workspace() {
+	local workspace_dir="${PWD}"
+	if [ ! -z "${1}" ]; then
+		workspace_dir="${1}"
+		if [ ! -d "${1}" ]; then
+			if ! mkdir -p "${1}"; then
+				show_usage "Cannot create directory ${1}"
+				return 1
+			fi
+		fi
+	fi
+
+	pushd "${workspace_dir}" >>/dev/null
```

Comment:
I think it's customary to use `>/dev/null`. There's no technical value in appending to it ;-).